Src/StdLib/Lib/subprocess.py

```diff
@@ -1510,13 +1190,16 @@ def _dup2(a, b):
 
         def _handle_exitstatus(self, sts, _WIFSIGNALED=os.WIFSIGNALED,
                 _WTERMSIG=os.WTERMSIG, _WIFEXITED=os.WIFEXITED,
-                _WEXITSTATUS=os.WEXITSTATUS):
+                _WEXITSTATUS=os.WEXITSTATUS, _WIFSTOPPED=os.WIFSTOPPED,
```

Comment:
Do we have all these properties on the os module?

Reply:
No, but they're in an unsupported codepath right now (if not windows and not posix).